driver-sync/src/main/com/mongodb/client/MongoCollection.java

```diff
@@ -1693,6 +1694,84 @@ BulkWriteResult bulkWrite(ClientSession clientSession, List<? extends WriteModel
      */
     void drop(ClientSession clientSession, DropCollectionOptions dropCollectionOptions);
 
+    /**
+     * Create an Atlas Search index for the collection.
+     *
+     * @param indexName  the name of the search index to create.
+     * @param definition the search index mapping definition.
+     * @return the search index name.
+     * @mongodb.server.release 7.0
+     * @mongodb.driver.manual reference/command/createSearchIndexes/ Create Search indexes
+     * @since 4.11
+     */
+    String createSearchIndex(String indexName, Bson definition);
```

Comment:
Does creation / listing / dropping also support sessions?

Reply:
Currently, it is not possible to use the new search index commands within transactions. While this fact is not explicitly mentioned in the specification, other drivers have also not implemented support for them. However, it is not breaking to add support for them later if necessary.